thoth/common/openshift.py

```diff
@@ -1679,14 +1679,16 @@ def schedule_adviser(
 
         workflow_parameters = self._assign_workflow_parameters()
 
-        return self._schedule_workflow(
+        _ =  self._schedule_workflow(
```

Comment:
No need to do assignment - the return value will be simply discarded.

Reply:
isn't it a better practice according to some linters not to ignore a return value?

Reply:
Which ones? I don't see any reason why they would require to do that. A lot of return values of function calls are simply ignored.

Reply:
Not sure. Just know that I've seen a message like that somewhere :D I will get rid of that, not a problem :)